### ISSUE TITLE
Features present on caniuse but not present in data.json

### ISSUE THREAD

**Issue by dfabulich** [Fyrd/caniuse#6814]
https://caniuse.com/mdn-javascript_operators_spread

This feature appears on caniuse.com, but the string `javascript_operators_spread` doesn't appear in any file in caniuse Github commit d5ef0b9a0a7aa88cb2c2ef919fb20b1654179bc8.

**Comment by dfabulich** [comment]
It doesn't appear in https://caniuse.com/ciu/index either!

**Comment by Fyrd** [comment]
That's correct, there's a large number of features provided by the [MDN BCD](https://github.com/mdn/browser-compat-data) data. Because they're already part of another repo they only appear as part of the search and not on other places like the index. You can identify these features by the footer of the support table and by the "mdn-" prefix as part of the identifier.

**Comment by dfabulich** [comment]
Where can I find `usage_perc_y` for these features?

**Comment by Fyrd** [comment]
Not currently available as part of the exported data I'm afraid, just calculated for the site.

**Comment by dfabulich** [comment]
This seems like a serious bug. data.json is just missing a bunch of features; the fact that they came from MDN doesn't mean that the features shouldn't be in data.json.

Would you kindly export that data? Even just putting in a separate file would be fine.
